tee-worker/litentry/core/assertion-build/src/dynamic/contracts/token_holding_amount/TokenMapping.sol

```diff
@@ -73,352 +73,273 @@ import { Usdd } from "./erc20/Usdd.sol";
 contract TokenMapping is TokenQueryLogic {
 	constructor() {
 		// btcs
-		tokenNames["btcs"] = Btcs.getTokenName();
 		tokenRanges["btcs"] = Btcs.getTokenRanges();
-		tokenNetworks["btcs"] = Btcs.getTokenNetworks();
+		for (uint8 i = 0; i < Btcs.getTokenInfo().length; i++) {
+			tokenInfo["btcs"].push(Btcs.getTokenInfo()[i]);
+		}
```

Comment:
maybe no need for-each, just use tokenInfo["btcs"] = Btcs.getTokenInfo(), would that work?

Reply:
Wouldn't,[an error ](https://stackoverflow.com/questions/71984832/unimplementedfeatureerror-copying-of-type-struct-contractname-structname-memo)will occur.

Reply:
The simplest way to handle the structure array copy is to do it like this, other methods will be more complicated.

Reply:
Struct array copying is used elsewhere, push must be used to handle it, otherwise it cannot be used(an error as above), so for-each is inevitable. :(